### DISCUSSION TITLE
Mitre mapping in cases

### DISCUSSION THREAD

**Discussion by OzzyKampha** [Security-Onion-Solutions/securityonion#8662] [Ideas] [answered]
I would love a feature there i can choose tags from the Mitre matrix in cases.

**Comment by dougburks** [comment, marked as answer]
You can customize the tag presets as shown here:
https://docs.securityonion.net/en/2.3/soc-customization.html#cases

**Reply by OzzyKampha** [reply]
Thx, i will check that out
